alembic/env.py

```diff
@@ -0,0 +1,78 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+import models
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+if config.config_file_name is not None:
+    fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = [models.BaseClass.metadata]
```

Comment:
The `target_metadata` is set to `[models.BaseClass.metadata]`, which is a list. Alembic expects `target_metadata` to be a single MetaData object, not a list. You should ensure that `models.BaseClass.metadata` is the correct MetaData object and assign it directly to `target_metadata` without wrapping it in a list.